src/components/override-sdk/infra/Assignment/Assignment.tsx

```diff
@@ -154,6 +178,40 @@ export default function Assignment(props) {
     setErrorSummary(true);
   }
 
+  function isServiceShuttered() {
+    const featureID = 'ChB';
+    const featureType = 'Service';
+
+    const parameters = new URLSearchParams(
+      `{FeatureID: ${featureID}, FeatureType: ${featureType}}`
+    );
+
+    const url = `${shutterPageUrl}?dataViewParameters=${parameters}`;
+
+    const invokePromise = getPConnect().getActionsApi().invoke(url, 'GET');
+
+    invokePromise
+      .then(resp => {
+        const isShuttered = resp.data.Shuttered;
+        setServiceShuttered(isShuttered);
+      })
+      .catch(err => {
+        // eslint-disable-next-line no-console
+        console.error(err);
+      });
+  }
+
+  // Runs the is service shuttered function and sets the shutter rest api url when the view changes
+  useEffect(() => {
+    getSdkConfig().then(sdkConfig => {
+      const url = new URL(
+        `${sdkConfig.serverConfig.infinityRestServerUrl}/app/chb-dev/api/application/v2/data_views/D_ShutterLookup`
```

Comment:
Hi @nickwhm  Please don't hardcode the path as it won't work on different environment! This needs to be dynamic. Thanks